Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -2228,6 +2228,15 @@
     normalState: normal
     denyState: deny
     ejectDelay: 2
+  - type: Sprite
```

Comment:
you should indicate that this is an altered upstream file with a comment (# imp)

Reply:
the meta.json should have this? if that didnt get pushed im going to be annoyed

Reply:
this specific comment was on the syndiejuice vendor. this is an upstream prototype you are modifying, so it should be indicated as such, so that the changes aren't lost in the case of upstream modifying this prototype